### PR TITLE
Update EIP-4675: Fix SPDX license identifier

### PR DESCRIPTION
**Pull request by Pandapip1**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by Pandapip1** [comment]
Bump.

**Comment by lightclient** [comment]
Same issue as mentioned here: https://github.com/ethereum/EIPs/pull/5227#issuecomment-1195604313

**Comment by PowerStream3604** [comment]
Can you elaborate on the reason for changing the license?
Unless there is a particular reason, I'm not opposed to changing it.
@Pandapip1

**Comment by Pandapip1** [comment]
All assets should be CC0. Exceptions can be made, but in general, assets being the same license as the EIP itself is the policy.

This is primarily to help aid the adoption of your EIP so that people aren't concerned about using your assets due to unfamiliar or incompatible licenses.

**Comment by Pandapip1** [comment]
@PowerStream3604 please approve this PR

**Review by PowerStream3604 — DISMISSED** [review]
Approve changing the License

**Comment by Pandapip1** [comment]
@SamWilsn I request a manual merge.

**Comment by Pandapip1** [comment]
Still an issue.

File: assets/eip-4675/contracts/math/SafeMath.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: CC0-1.0
```
Comment (SamWilsn) [inline assets/eip-4675/contracts/math/SafeMath.sol:1]: Correct me if I'm wrong, but this looks like OpenZeppelin. You cannot change the license of code you do not own.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

File: assets/eip-4675/contracts/math/SafeMath.sol (hunk 1 above)
Comment (Pandapip1) [reply assets/eip-4675/contracts/math/SafeMath.sol:1]: This is solidity v0.8.0... why is OZ SafeMath used anyway???

File: assets/eip-4675/contracts/ERC20Token.sol [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: CC0-1.0
```
Comment (xinbenlv) [inline assets/eip-4675/contracts/ERC20Token.sol:1]: i think we shall ask the author to grant it with new license....

Comment (Pandapip1) [reply assets/eip-4675/contracts/ERC20Token.sol:1]: We did ask, and the author approved.

File: assets/eip-4675/contracts/math/SafeMath.sol (hunk 1 above)
Comment (Pandapip1) [reply assets/eip-4675/contracts/math/SafeMath.sol]: Problem solved. Needs manual merge now.

**Comment by Pandapip1** [comment]
Needs manual merge due to deleting a file.

**Pull request merged**